readthedocs/projects/forms.py

```diff
@@ -175,6 +175,29 @@ class Meta(object):
         widget=forms.Textarea,
     )
 
+    def clean_language(self):
+        language = self.cleaned_data['language']
+        project = self.instance
+        if project:
+            msg = _(
+                'There is already a "{lang}" translation '
+                'for the {proj} project.'
+            )
+            format_msg = msg.format(lang=language, proj=project.slug)
+            if project.translations.filter(language=language).exists():
+                raise forms.ValidationError(format_msg)
+            main_project = project.main_language_project
+            if main_project:
+                if main_project.language == language:
+                    raise forms.ValidationError(format_msg)
+                siblings = (main_project.translations
+                            .filter(language=language)
+                            .exclude(pk=project.pk)
+                            .exists())
+                if siblings:
+                    raise forms.ValidationError(format_msg)
```

Comment:
At this point, the `project` in the message, shouldn't be `main_project`?

Reply:
I'm not sure, all the messages here point to the current project. Should I change this?

Reply:
The question is, mentioning `project` will the messsage still be valid?

Which is the case of this validation error?

Reply:
I take this from https://github.com/rtfd/readthedocs.org/pull/3790#discussion_r174978992

I think is kind of unnecessary to show the project here, right? Maybe just `There is already a "{lang}" translation for this project.` is enough, what do you think?

Reply:
I think mentioning the project is a good idea, I'm +1 on altering the message to reference `main_project`.